### ISSUE TITLE
🚀 Pre-release master -> staging_KobayashiMaru5

### ISSUE THREAD

**Issue by YuryHrytsuk** [ITISFoundation/osparc-simcore#5148]
### What kind of pre-release?

master branch

### Sprint Name

KobayashiMaru

### Pre-release version

5

### Commit SHA

`09dd712f6c66bf2336f6321a6bd1e0430c7de7eb`

### Did the commit CI suceeded?

- [x] The commit CI succeeded.

### Motivation

Weekly staging release

### What Changed

* https://github.com/ITISFoundation/osparc-simcore/pull/5143 by @jsaq007
* https://github.com/ITISFoundation/osparc-simcore/pull/5141 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/5145 by @ignapas
* https://github.com/ITISFoundation/osparc-simcore/pull/5122 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/5135 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/5147 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/5112 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/5151 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/5150 by @matusdrobuliak66
* https://github.com/ITISFoundation/osparc-simcore/pull/5153 by @dependabot
* https://github.com/ITISFoundation/osparc-simcore/pull/5155 by @sanderegg
* https://github.com/ITISFoundation/osparc-simcore/pull/5156 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/5158 by @GitHK
* https://github.com/ITISFoundation/osparc-simcore/pull/5111 by @jsaq007

### Devops check ⚠️ devops

- [x] Deploy RabbitMQ to sim4life.io before starting to release (USE PROPER PASSWORD)
- [x] Move NIH STAG Rabbit MQ to simcore rabbit container
- [x] Introduce user-defined certificates --> Won't do this time.

### Summary 📝

- [x] `` make release-staging name=<sprint_name> version=<version> git_sha=<commit_sha>``
   - `https://github.com/ITISFoundation/osparc-simcore/releases/new?prerelease=1&target=<commit_sha>&tag=staging_<sprint_name><version>&title=Staging%20<sprint_name><version>`
- [x] Draft [pre-release](https://github.com/ITISFoundation/osparc-simcore/releases)
- [x] Announce (add redis key ```maintenance``` in every concerned deployment)
  ```json
  {"start": "2023-02-01T12:30:00.000Z", "end": "2023-02-01T13:00:00.000Z", "reason": "Release ResistanceIsFutile9 "}
  ```
- [ ] Announce release in Mattermost
  ```
  :loud_sound:  Maintenance scheduled for **NAMED_DAY DD. MM from START_TIME - END_TIME**.
  =========================================================================

  @all Be aware that you will automatically be logged out and your projects stopped and saved during the maintenance time. Affected:
  *   [https://staging.osparc.io](https://staging.osparc.io/)
  *   [https://https://staging.s4l-lite.io/](https://https://staging.s4l-lite.io//)

  and on premises:
  *   [https://osparc-staging.speag.com](https://osparc-staging.speag.com/)
  *   [https://tip-staging.speag.com](https://tip-staging.speag.com/)
  *   [https://s4l-staging.speag.com](https://s4l-staging.speag.com/)
  *   [https://s4l-lite-staging.speag.com](https://s4l-lite-staging.speag.com/)


  Reason: Scheduled staging-release of STAGING_NAME_AND_VERSION.

  Thanks for your understanding and sorry for the inconveniences,

  Your friendly oSparc Team

  ```

### Releasing

- [x] Release (release draft)
- [x] Check Release CI
- [x] Check hanging sidecars. Helper command to run in director-v2 CLI `simcore-service-director-v2 close-and-save-service <uuid>`
- [x] Check deployed
  - [x] aws deploy
  - [ ] dalco deploy
- [x] Delete announcement
- [ ] Check e2e runs
- [ ] Announce
``` md
https://github.com/ITISFoundation/osparc-simcore/releases/tag/staging_<sprint_name><version>
```

**Comment by matusdrobuliak66** [comment]
:warning: for Matus :warning:

Because of TTL change in rabbitmq queues, it might be needed to delete these queues manually:
- `io.simcore.service.tracking`
- `io.simcore.service.wallets`
- `io.simcore.service.wallets-credit-limit-reached`
- `io.simcore.services.instrumentation`

Either delete them preventively before releasing, or if some services have problem to start just delete the queues and restart the service!
